### PR TITLE
Blog post for v2.10.1 patch release

### PR DESCRIPTION
**Pull request by scotthmurray**
For your review... Please double check that I'm using the appropriate Markdown for the blog generator (i.e. appropriate heading levels, etc.).

### CODE REVIEW

**Comment by lkhrs** [comment]
Hi Scott! The headings need to go down one level, because there's already an H1 on the page when built (the title).

**Comment by scotthmurray** [comment]
@lkhrs Updated, thank you!

File: site/blog/2022/v2.10.1.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Hi, data friends!
+
+It’s time for a [new patch release](https://github.com/saulpw/visidata/releases/tag/v2.10.1) with some important fixes and a few minor improvements.
+
+### Fixes
+
+- Macros weren’t working; now they are!
+- Repeat commands were broken, and now are fixed.
+- The Guix build is now working again.
```
Comment (anjakefala) [inline site/blog/2022/v2.10.1.md:18]: I would phrase this as "changes necessary for the Guix build have been merged"? Or something like that! It is not that the Guix build was "broken", but that the Guix build maintainer could not build VisiData.

File: site/blog/2022/v2.10.1.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: v2.10.1
+date: 2022-09-14
+tags:
+    - releases
+---
```
Comment (anjakefala) [inline site/blog/2022/v2.10.1.md:6]: Headers are great =)

File: site/blog/2022/v2.10.1.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+## VisiData v2.10.1 Released
```
Comment (anjakefala) [inline site/blog/2022/v2.10.1.md:8]: The title is usually something like:

```
## [v2.10.1](https://github.com/saulpw/visidata/releases/tag/v2.10.1) (2021-09-14)
```

Where the date is the date of the release. That link will not work for now, but will work once we officially release and create the tag on VisiData.

**Comment by anjakefala** [comment]
Thanks for putting this together!

**Comment by scotthmurray** [comment]
Thank you, @anjakefala! I believe I've accommodated those comments now.

**Review by saulpw — APPROVED** [review]

**Pull request merged**